docs/reference/requirement-sets/powerpoint-api-requirement-sets.md

```diff
@@ -0,0 +1,58 @@
+---
+title: PowerPoint JavaScript API requirement sets
+description: ''
+ms.date: 07/17/2019
+ms.prod: powerpoint
+localization_priority: Normal
+---
+
+# PowerPoint JavaScript API requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+The following table lists the PowerPoint-specific requirement sets, the Office host applications that support those requirement sets, and the build versions or availability date.
+
+|  Requirement set  |  Office on Windows<br>(connected to Office 365 subscription)  |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web |
+|:-----|-----|:-----|:-----|:-----|:-----|
+| PowerPointApi 1.1 | Version 1903 (Build 11425.20204) or later | 2.24 or later | 16.24 or later | May 2019 |
+
```

Comment:
Seems like this section that appears in other Requirement Sets articles (e.g., Excel, Word) contains useful/relevent info:

![image](https://user-images.githubusercontent.com/3019529/61401058-772c8080-a886-11e9-936d-1c0774a43ad8.png)

Perhaps consider adding that section here, after the table?  (could just copy/paste that section from the [Excel requirement sets overview article](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets)

Reply:
For consistency, suggest making this same addition to the [onenote-api-requirement-sets.md](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/onenote-api-requirement-sets) article.

Reply:
I think I'll leave this out of the OneNote article, since only OneNote on the Web is supported and that doesn't really have build numbers.

Reply:
Yes, good call.